data/ATRP_2020/data.csv

```diff
@@ -1,2758 +1,2758 @@
-taxon_name,growth_form_icons,plant_growth_form,growth_habit,parasitic,life_history,regen_strategy,growth_medium,sex_type,,
```

Comment:
did you mean to change this file

Reply:
Yes - here or elsewhere. There are two "blank" columns at the end that must have some space/character, so they are read in and because they are unnamed they lead to a warning message each time the file is read in. So I just deleted the columns.